src/components/IconButton/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { IconButtonProps } from '../../types/props';
+import './IconButton.scss';
+
+const IconButton = ({ name }: IconButtonProps): JSX.Element => {
+  return (
+    <a className="download-link" title={`Download ${name}`}>
```

Comment:
Might be worth looking at https://ant.design/components/button we should probably use the components from there.